server/src/main/java/org/apache/iotdb/db/service/thrift/impl/InternalServiceImpl.java

```diff
@@ -382,6 +375,57 @@ public TSStatus deleteRegion(TConsensusGroupId tconsensusGroupId) throws TExcept
     }
   }
 
+  @Override
+  public TMigrateRegionsResp migrateRegions(TMigrateRegionsReq req) throws TException {
```

Comment:
need  we call addConsensusGroup or removeConsensusGroup？

Reply:
This service is for changing peers in the consensus group. When migrating, config nodes will call one existing node which is still in the group to use this service. Afterwards, we can add other services to execute addConsensusGroup or removeConsensusGroup on the new node or old node.